src/components/carousel/index.jsx

```diff
@@ -231,7 +231,9 @@ const Carousel = ({
 			setIsAutoplaying(false);
 		} else {
 			setSlide(slide + 1);
-			setPosition(position - 100 / slidesToShowInView);
+			const slideOffset =
+				carouselElement.current.querySelector(".c-carousel__slide:nth-of-type(2)")?.offsetLeft || 0;
```

Comment:
I both don't like the querySelector (should use references in React) and I am curious about the "2" here.  Is this ONLY ever the second item?

Reply:
this is how it's done for the next and previous logic. the current logic is not working. 

We are using a reference with carouselElement. `nth-of-type` is for calculating how much to slide

Reply:
Oh, I see, it's just grabbing the second one and using it's left position to determine the shifting value.

I can accept this, but as a grander issue, I think I'll note this as a wiggle week change or something.  It's probably not breaking, but more of an implementation detail.